charts/kubefed/README.md

```diff
@@ -121,6 +121,10 @@ chart and their default values.
 | controllermanager.clusterHealthCheckTimeout          | Duration after which the cluster health check times out.                                                                                                                     | 3s                              |
 | controllermanager.syncController.adoptResources      | Whether to adopt pre-existing resource in member clusters.                                                                                                        		        | Enabled                         |
 | controllermanager.service.labels                     | Kubernetes labels attached to the controller manager's services                                                                                                       		    | {}                              |
+| controllermanager.certManager.enabled             | Specifies whether to enable the usage of the cert-manager for the certificates generation.                                                                                      | false                           |
+| controllermanager.certManager.organizations       | Specifies the list of organizations to include in the cert-manager generated ca certificate.                                                                                    | []                              |
+| controllermanager.certManager.commonName     | Specifies the CN value for the cert-manager generated ca certificate.                                                                                                                | ca.webhook.kubefed              |
+| controllermanager.certManager.dnsNames       | Specifies the list of subject alternative names for the cert-manager generated ca certificate.                                                                                       | ["ca.webhook.kubefed"]          |
```

Comment:
Can we somehow incorporte into these names the fact that these will only get applied to single Issuer configuration - `{{ .Release.Name}}-root-certificate`

Reply:
I added it under the rootCertificate element to be more clear about it.